Content.Server/Spreader/KudzuSystem.cs

```diff
@@ -51,6 +51,9 @@ private void OnKudzuSpread(EntityUid uid, KudzuComponent component, ref SpreadNe
         if (component.GrowthLevel < 3)
             return;
 
+        if (component.SpreadRadius == 0)
+            return;
+
```

Comment:
This is doubling smokecomponent so these need to be made similar.

Reply:
I've looked at how SmokeSystem works, but I don't think that approach will work here. The smoke gradually loses density, creating copies. Kudza is supposed to capture and hold a certain area. In the current implementation, if you destroy a part of the kudza, it will eventually take over the same area again and stop.